core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/KamonRecorder.scala

```diff
@@ -87,16 +89,32 @@ object KamonRecorder extends MetricRecorder with KamonMetricNames with SLF4JLogg
         `actionName` -> action,
         `actionKind` -> kind,
         `actionMemory` -> memory)
+    private val namespaceActivationsTags =
+      Map(
+        `actionNamespace` -> namespace,
+        `initiatorNamespace` -> initiator,
+        `actionKind` -> kind,
```

Comment:
For namespace level metrics we can drop the `actionKind` and `actionMemory` tags